stories/Molecules/Navbar/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./pagination.scss";
+
+export const Pagination = ({ text, text2 }) => {
+  return (
+    <ul className="pagination__wrapper">
```

Comment:
@harishkumar-srijan it should be pagination-wrapper class it is not element on any block